contentcuration/contentcuration/frontend/shared/vuex/session/index.js

```diff
@@ -75,5 +75,13 @@ export default {
       currentUser = { ...currentUser, first_name, last_name };
       context.commit('SET_CURRENT_USER', currentUser);
     },
+    logTagEvent(context, params) {
+      window.dataLayer.push({
+        studioVersion: window.studioVersion,
+        pageCategory: window.pageCategory,
```

Comment:
The above two keys do not need to be repeated whenever something is added to the data layer. Could you push them into the `dataLayer` in the template? We actually don't want `studioVersion` at all if you would like to remove it. I also realized, not all the templates are `pushing`, but some are setting the dataLayer variable. I can fix that

Reply:
Oh, I thought I had changed them all to push.

Reply:
Can update to not set these events on every push, certainly!